src/services/WolfDen.Application/Requests/Commands/Employees/AddEmployee/AddEmployeeCommand.cs

```diff
@@ -10,4 +10,8 @@ public class AddEmployeecommand : IRequest<int>
 
 
     }
+  /*  public class Errors
+    {
+        public string[] _ { get; set; }
```

Comment:
remove unwanted code

Reply:
done
used it to check how the code will return the error code